app/containers/TracksContainer/index.js

```diff
@@ -1,5 +1,168 @@
-import React from 'react';
+import React, { memo, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { debounce, get, isEmpty } from 'lodash';
+import styled from 'styled-components';
+import { injectIntl } from 'react-intl';
+import { connect } from 'react-redux';
+import { injectSaga } from 'redux-injectors';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { Input, Card, Skeleton } from 'antd';
+import { tracksContainerCreators } from './reducer';
+import tracksContainerSaga from './saga';
+import T from '@components/T';
+import For from '@app/components/For';
+import If from '@app/components/If';
+import TrackComponent from '@app/components/TrackComponent/index';
+import { selectTrackName, selectTracksContainer, selectTracksData, selectTracksLoading } from './selectors';
 
-export default function TracksContainer() {
-  return <div> TracksContainer page </div>;
+const { Search } = Input;
+
+const TitleCard = styled(Card)`
+  && {
+    margin: 0.5rem 0;
+    color: ${props => props.color};
+    width: 100%;
+    max-width: ${props => props.maxWidth}rem;
+  }
+`;
+
+const StyledSearchContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+`;
+
+const Container = styled.div`
+  && {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    width: 100%;
+    margin: 0 auto;
+    padding: ${props => props.padding}rem;
+  }
+`;
+
+const TrackGrid = styled.div`
+  display: flex;
+  gap: 2rem;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 auto;
+`;
+
+export function TracksContainer({
+  maxWidth,
+  padding,
+  tracksData,
+  trackName,
+  intl,
+  dispatchTrackNames,
```

Comment:
can we name this a little better ?
dispatchRequestTracksData

Reply:
updated the changes